core/services/machine_configurator.rb

```diff
@@ -137,6 +139,16 @@ def install_tgz_chef(machine, chef_version, architecture, logger)
     Result.error('Unable to install appimage chef!')
   end
 
+  def check_and_install_tar(machine, logger)
+    result = sudo_exec(machine, 'which tar', logger)
+    if result.success?
+      result
+    else
+      logger.info('Tar is not found. Downloading...')
+      sudo_exec(machine, 'yum install -y tar', logger)
```

Comment:
Current fix is only usefull in RHEL-based distributions. Please add call to the `apt-get` to enable support for Debian-based distributions.